<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="dataText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dataComboBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Text</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hex</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="logText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>581</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="dataLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string> Write Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="logLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string> Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="repeatButton">
    <property name="geometry">
     <rect>
      <x>464</x>
      <y>150</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Repeatedly </string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Connection"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Log"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen_Connection">
   <property name="text">
    <string>Open Connection</string>
   </property>
  </action>
  <action name="actionSave_Log">
   <property name="text">
    <string>Save Log</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
